src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -182,14 +183,14 @@ public static TableModel tableModelFromTable(DatasetTable datasetTable) {
     public static Column columnModelToDatasetColumn(ColumnModel columnModel) {
         return new Column()
                 .name(columnModel.getName())
-                .type(columnModel.getDatatype())
+                .type(columnModel.getDatatype().toString())
                 .arrayOf(columnModel.isArrayOf());
     }
 
     public static ColumnModel columnModelFromDatasetColumn(Column tableColumn) {
```

Comment:
This can be changed to a member function on `Column` and the `Column` parameter can be eliminated.

Reply:
which column parameter do you mean by the Column parameter can be eliminated? Which function do you want changed to a member function?